ckanext/pdfview/plugin.py

```diff
@@ -46,6 +49,10 @@ def can_view(self, data_dict):
         proxy_enabled = p.plugin_loaded('resource_proxy')
         same_domain = datapreview.on_same_domain(data_dict)
 
+        pdf_size = resource.get('Size', 0)
+        if pdf_size > self.custom_max_size:
+            h.flash_notice(_('This PDF is too big for preview. Downloading the file is suggested'))
```

Comment:
```suggestion
            h.flash_notice(_('This PDF is too large to be previewed in the browser. Downloading the file is suggested'))
```
I suggest we use the word "large" instead of "big and let them know it can't be previewed in the browser explicitly.

Reply:
Good idea!Thanks